docs/source/modules.rst

```diff
@@ -232,6 +232,12 @@ analyses/waterfacilityrestoration
 ..  autoclass:: waterfacilityrestoration.waterfacilityrestoration.WaterFacilityRestoration
     :members:
 
+analyses/wfnfunctionality
+=================================
+..  autoclass:: wfnfunctionality.wfnfunctionality.WfnFunctionality
+    :members:
+
+
 models
```

Comment:
Not related to this PR, but can you remove the extraneous reference here?

Reply:
Do you mean the empty space?

Reply:
There is a dangling "models" on line 241

Reply:
ahhh, you are right. analyses has the same thing, my mistake.